src/Tickets/Commerce/Order_Modifiers/Modifiers/Modifier_Abstract.php

```diff
@@ -182,29 +182,32 @@ public function get_modifier_by_applied_to( array $applied_to, array $params = [
 	}
 
 	/**
-	 * Finds a modifier by its display name.
+	 * Get modifiers along with their applied to meta key.
 	 *
-	 * @since 5.18.0
+	 * @since TBD
 	 *
-	 * @param array $search Parameters to search Order Modifiers by.
+	 * @param array $params               Additional parameters to filter the results.
+	 * @param bool  $with_applied_to_meta Whether to include the applied to meta key.
 	 *
-	 * @return array The modifier data.
+	 * @return array The modifiers that were found.
 	 */
-	public function find_by_search( array $search ): array {
-		return $this->repository->search_modifiers( $search );
+	public function get_modifiers( array $params = [], bool $with_applied_to_meta = true ): array {
+		return $this->repository->get_modifiers( $params, $with_applied_to_meta );
 	}
 
 	/**
-	 * Finds a modifier by the given search criteria and returns the count.
+	 * Finds a modifier by its display name.
 	 *
-	 * @since TBD
+	 * @since 5.18.0
+	 * @deprecated TBD Use `get_modifiers` instead.
 	 *
 	 * @param array $search Parameters to search Order Modifiers by.
 	 *
-	 * @return int The count of the modifiers found.
+	 * @return array The modifier data.
 	 */
-	public function find_count_by_search( array $search ): int {
-		return $this->repository->get_search_count( $search );
+	public function find_by_search( array $search ): array {
```

Comment:
We don't need to deprecate this, as it's being added in this release. We can just remove it.